pom.xml

```diff
@@ -125,30 +168,34 @@
             <dependency>
                 <groupId>com.github.spullara.mustache.java</groupId>
                 <artifactId>compiler</artifactId>
-                <version>0.8.17</version>
+                <!-- Upgrading to 0.9.0 causes an ArrayIndexOutOfBoundsException for some reason
+                     Failed to execute goal org.apache.maven.plugins:maven-plugin-plugin:3.2:helpmojo 
+                     (help-goal) on project retirejs-maven-plugin: Execution help-goal of goal 
+                     org.apache.maven.plugins:maven-plugin-plugin:3.2:helpmojo failed: 9578 -->
+                <version>0.8.18</version>
             </dependency>
 
             <!-- JSON parser use by the core -->
 
             <dependency>
                 <groupId>org.json</groupId>
                 <artifactId>json</artifactId>
-                <version>20140107</version>
+                <version>20180130</version>
             </dependency>
 
             <!-- ZAP stuff -->
 
             <dependency>
                 <groupId>org.zaproxy</groupId>
                 <artifactId>zap</artifactId>
-                <version>2.5.0</version>
+                <version>2.7.0</version>
```

Comment:
While not required it's probably better to update the add-on manifest (or update just to 2.6.0, to avoid using 2.7.0 features not available in 2.6.0):
https://github.com/h3xstream/burp-retire-js/blob/cce4373d5b8ff1487f03079bc1ab3e3d9ae9b9f6/retirejs-zap-plugin/src/main/resources-filtered/ZapAddOn.xml#L17